index.html

```diff
@@ -1,47 +1,128 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Cookie Store</title>
+<!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
+<!-- Optional theme -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+
+<!-- Latest compiled and minified JavaScript -->
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+
+<title>Cookie Store</title>
+
+<link rel="stylesheet" href="css/styles.css">
+
 </head>
 <body>
+<div class="container-fluid">
 
+	<div class="row" id = "banner" >
+		<div class="row"></div>
+		<div class="row"></div>
+		<div class="row"></div>
+		<div class="row"></div>
+		<div class="row"></div>
+		<div class="col-md-4 col-md-offset-4" id="cookiestore">
+			<div class = "text-center">
+				<h1>Cookie Store</h1>
+				<p>Buy our awesome cookies. All proceeds go to charity!</p>
+			</div>
+		</div>
+	</div>
+<div class="col-md-offset-5">
+	<h2>The Cookies</h2>
+</div>
+	
+<div id="products">
+	<div class="row">
+	<div class="col-md-offset-4">
+	<div class="col-md-3 push-md-2">	
+		<h3>Thin Mint Cookies</h3>
+		<h4 id="new" class="col-xs-1"><span class="label label-danger">NEW</span></h4>
+		<div class="row"></div>
+		<p>Tasty mint chocolate cookies</p>			
+	</div>
+	<div class="col-md-5 pull-md-2">	
+		<img src="img/mint.png" class="img-circle">	
+	</div>
+	</div>
+	</div>	
 
-<h1>Cookie Store</h1>
-
-<p>Buy our awesome cookies. All proceeds go to charity!</p>
+	<div class="row">
+	<div class="col-md-offset-4">
+	<div class="col-md-4 push-md-3">
+		<img src="img/peanut.png" class="img-circle">
+	</div>
+	<div class="col-md-3 pull-md-3">
+		<h3>Peanut Butter Cookies</h3>
+		<p>Yummy peanut buttery goodness!</p>	
+	</div>
+	</div>
```

Comment:
A couple of things are happening here.
- On line 54, you're moving the column to an offset of 4 but it doesn't have a size. This parent div will limit the width for the cookie `h3`, `p`, and `img` tags
- You should not change the order of the h3, p, and img tags. Semantically the img tag belongs below the h3 heading
- The way you wrapped the img tag and the h3 and p tag is good, and using the push and pull classes is also good. The issue here is that those two column don't add up to 12. You hacked to be centered by only using 7 columns, and having the parent offset by 4.

You should fix the other cookie product listings as well.